pages/common/vue-prototyping.md

```diff
@@ -0,0 +1,13 @@
+# vue-prototyping
```

Comment:
This page contains multiple subcommands. Maybe we could split `serve` and `build` into their own sub-pages?

Reply:
Thanks for your suggestion, applying changes.

Reply:
Changes commited to `patch-4`.